packages/ui/src/composites/wui-icon-box/index.ts

```diff
@@ -40,7 +42,7 @@ export class WuiIconBox extends LitElement {
 
     return html`
       <div class="wui-overlay-default ${classMap(classes)}" style=${bgColorStyle}>
```

Comment:
I think we might be able to get rid of this div now via using:

```
this.classList = classMap(classes)
this.style.cssText = bgColorStyle
```

and styling :host directly (with display block)